spec/cli/commands/ip_spec.rb

```diff
@@ -5,62 +5,119 @@
 describe Ronin::CLI::Commands::Ip do
   include_examples "man_page"
 
-  let(:ipcmd) { Ronin::CLI::Commands::Ip.new}
+  subject { Ronin::CLI::Commands::Ip.new }
```

Comment:
You can omit `subject { }` entirely since we're not initializing the class with any special arguments. By default `subject` is just `subject { described_class.new }`.

Reply:
Very cool!  Thanks for the tip.  I'll fix it.